conan/tools/system/package_manager.py

```diff
@@ -234,7 +237,7 @@ def __init__(self, conanfile, arch_names=None):
 
         self._arch_separator = ":"
 
-    def install(self, packages, update=False, check=True, recommends=False):
+    def install(self, packages, update=False, check=True, host_package=True, recommends=False):
```

Comment:
```suggestion
    def install(self, packages, update=False, check=True, recommends=False, host_package=True):
```
The likelihood to break existing code is lower if new arguments are added at the end.